src/development/tools/devtools/release-notes/release-notes-2.18.0-src.md

```diff
@@ -0,0 +1,88 @@
+  [#4503](https://github.com/flutter/devtools/pull/4503)
+- Add offline support for raster stats -
+  [#4491](https://github.com/flutter/devtools/pull/4491)
+- Add 'Rendering time' column to Raster Metrics tab -
+  [#4474](https://github.com/flutter/devtools/pull/4474)
+
+# CPU profiler updates
+
+- Fix crash when an empty frame is filtered -
+  [#4502](https://github.com/flutter/devtools/pull/4502)
+- Fix bugs in CPU profile trees -
+  [#4413](https://github.com/flutter/devtools/pull/4413)
+- UI Cleanup - [#4404](https://github.com/flutter/devtools/pull/4404)
+
+# Memory updates
+
+- UI cleanup - [#4520](https://github.com/flutter/devtools/pull/4520)
+- Allocation tracing on multiple isolates is now supported -
+  [#4538](https://github.com/flutter/devtools/pull/4538)
+- Clear button added to allocation tracing tab -
+  [#4534](https://github.com/flutter/devtools/pull/4534)
+- Fix sorting column not being correctly updated -
+  [#4532](https://github.com/flutter/devtools/pull/4532)
+- Add Profile and Allocation Tracing sub-tabs -
+  [#4523](https://github.com/flutter/devtools/pull/4523)
+- Implement snapshot visualization -
+  [#4473](https://github.com/flutter/devtools/pull/4473)
+- Implement comparing memory profiles -
+  [#4501](https://github.com/flutter/devtools/pull/4501)
+- Remove settings for displaying units -
+  [#4370](https://github.com/flutter/devtools/pull/4370)
+- Advanced memory behavior is now always on and no longer controlled by
+  settings - [#4370](https://github.com/flutter/devtools/pull/4370)
+- Remove android button from chart control pane. It can be turned on/off in
+  settings - [#4370](https://github.com/flutter/devtools/pull/4370)
+- Add Call Tree view - [#4403](https://github.com/flutter/devtools/pull/4403)
+- Persist profile sort order -
+  [#4405](https://github.com/flutter/devtools/pull/4405)
+
+# Debugger updates
+
+- Fix bug for file opener and search -
+  [#4525](https://github.com/flutter/devtools/pull/4525)
+- Fix the codeview's scrollable area -
+  [#4448](https://github.com/flutter/devtools/pull/4448)
+- Allow syntax highlighting on nested captures in parser -
+  [#4427](https://github.com/flutter/devtools/pull/4427)
+
+# Network profiler updates
+
+- When on the Network tab, network recordings now continue working after the app
+  hot restarts - [#4438](https://github.com/flutter/devtools/pull/4438)
+
+# Logging updates
+
+- Log messages from non-stdout sources are now shown -
+  [#4487](https://github.com/flutter/devtools/pull/4487)
+
+# VM tool updates
+
+- Add support for inspecting objects in an isolate's object store -
+  [#4513](https://github.com/flutter/devtools/pull/4513)
+- Enable object inspector / program explore view -
+  [#4514](https://github.com/flutter/devtools/pull/4514)
+- Add "Code Preview" section to Object Inspector views -
+  [#4447](https://github.com/flutter/devtools/pull/4447)
+- UI polish - [#4436](https://github.com/flutter/devtools/pull/4436)
```

Comment:
@bkonyi do you think we need to be adding VM tool updates to our release notes?

Reply:
Honestly, it's up to you since the features aren't enabled by default, but it might not hurt to include them.